scripts/eurostat/birth_death_migration/import_data.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pandas as pd
+
+
+def download_data():
+    """Downloads raw data from Eurostat website and stores it in instance
+    data frame.
+    """
+    raw_df = pd.read_table(DATA_LINK)
+    return raw_df
+
+
+def preprocess_data(raw_df):
```

Comment:
Since there is some amount of specific expectations in this func, can add a longer docstring about what preprocessing you are doing, and try to help differentiate preprocess from clean functions.